tasks/rogue/route/base.py

```diff
@@ -161,14 +167,15 @@ def clear_item(self, *waypoints):
     """
 
     def clear_elite(self, *waypoints):
+        # Use techniques before BOSS
+        logger.info('Trying to use technique')
+        UseTechniqueUI(self.config, self.device).use_technique_(1)
+
         logger.hr('Clear elite', level=1)
         waypoints = ensure_waypoints(waypoints)
         end_point = waypoints[-1]
         end_point.speed = 'run_2x'
 
-        # TODO: Use techniques before BOSS
-        pass
-
```

Comment:
Use techinique from multiple characters to empty technique points before boss

Reply:
I would refrain from using all technique pts for at least the first elite enemy since it might run out later.